pkg/identity/keystone/sync.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package keystone
+
+// SyncConfig contains a config for data synchronization between Keystone and Kubernetes
+type SyncConfig struct {
+	// TODO(mfedosin) Add more config options.
```

Comment:
There are several ideas what we can configure here:
1. What data we want to sync (this will be relevant when we also start syncing user roles).
2. Various black lists: by id, by name, by projects (for the roles), by domain.
3. namespace naming format. now it's just a project id, but I think this could be configurable to be more mnemonic.